types/src/front/lib/assistant.ts

```diff
@@ -204,6 +206,18 @@ export const MISTRAL_MEDIUM_MODEL_CONFIG: ModelConfigurationType = {
   shortDescription: "Mistral's smartest model.",
   supportsMultiActions: false,
 };
+export const MISTRAL_8X22B_MODEL_CONFIG: ModelConfigurationType = {
+  providerId: "mistral",
+  modelId: MISTRAL_8X22B_MODEL_ID,
+  displayName: "Mistral 8x22B",
+  contextSize: 64_000,
+  recommendedTopK: 16,
+  recommendedExhaustiveTopK: 56, // 28_672
```

Comment:
Are we sure about this?

Reply:
No, but I could not find the info on the doc :/

Reply:
We can likely push it a bit? what are other 64k model using?

Reply:
I added a commit!